sass/components/_single_image_view.scss

```diff
@@ -0,0 +1,38 @@
+.file-view--single {
+    .file__image {
+        position: relative;
+
+        .image-header {
+            button.post__embed-visibility {
+                width: 1.4rem;
+                height: 1.4rem;
+                color: #8c8c8d;
```

Comment:
this looks like we should use a variable here. @matthewbirtch could you help with the correct value?

Reply:
Thanks! Also curious about the border color (#e7e7e7). I used vars where it seemed like there was a valid option (i.e. the button), but didn't see anything that seemed to fit with those two.